fe/src/app/list/[advertId]/page.tsx

```diff
@@ -0,0 +1,45 @@
+import { BASE_API_URL, USER_JOBS_ROUTE } from "@/utils/urls";
+import { getAccessToken } from "@auth0/nextjs-auth0";
+import { Breadcrumbs } from "@/lib/materialTailwindExports";
+import Link from "next/link";
+import MyOffersList from "./components/MyOffersList";
+
+const getAdvertJobs = async (advertId: string) => {
+  const { accessToken } = await getAccessToken();
+  if (!accessToken) {
```

Comment:
It won't trigger because next js fires an error before this function is called if i try manually removing session id from my cookies while logged in:
![image](https://github.com/MatasAdmoneo/CarFans/assets/99315244/f3ca79d6-6443-4719-841e-205537ffbafd)

I provided the solution in your middleware and this if statement won't be needed anymore